compiler/testData/debug/stepping/when.kt

```diff
@@ -14,10 +14,11 @@ fun foo(x: Int) {
     }
 }
 
-fun box() {
+suspend fun box() {
```

Comment:
@pyos Intention of this change is not clear, could you clarify a bit?

Reply:
That was for checking that `suspend fun box` works, it's not needed in this test and should've been removed in the PR...